template.yaml

```diff
@@ -11,46 +11,49 @@ Globals:
     Timeout: 3
 
 Resources:
+  MSIS2Api:
+    Type: AWS::Serverless::Api
+    Properties:
+      StageName: dev
+      Domain:
+        DomainName: msis-api.dev.swx-trec.com
+        CertificateArn: !ImportValue accountCertificateArn
```

Comment:
This is what I found nice and didn't realize we could do with the outputs from other cloud formation stacks. I thought we had to hard-code them in after the output, but they are actually stored out in AWS for other stacks to reference and use.